Sources/TuistGenerator/Generator/WorkspaceGenerator.swift

```diff
@@ -116,13 +119,57 @@ final class WorkspaceGenerator: WorkspaceGenerating {
 
         try write(xcworkspace: xcWorkspace, to: workspacePath)
 
+        // SPM
+
+        try generatePackageDependencyManager(at: path,
+                                             workspace: workspace,
+                                             workspaceName: workspaceName,
+                                             graph: graph)
+
         // CocoaPods
 
         try cocoapodsInteractor.install(graph: graph)
 
         return workspacePath
     }
 
+    private func generatePackageDependencyManager(
+        at path: AbsolutePath,
+        workspace _: Workspace,
+        workspaceName: String,
+        graph: Graphing
+    ) throws {
+        let packages = try graph.targets.flatMap { try graph.packages(path: $0.path, name: $0.name) }
+        let hasPackages = !packages.isEmpty
+
+        guard hasPackages else {
+            return
+        }
+
+        let hasRemotePackage = packages.first(where: { package in
+            switch package.packageType {
+            case .remote: return true
+            case .local: return false
+            }
+        }) != nil
+
+        let packageResolvedPath = path.appending(component: "Package.resolved")
```

Comment:
Both packages are resolved, aren't they? I'd name them:
- `rootPackageResolvedPath`
- `workspacePackageResolvedPath`

Reply:
Yeah, should have given it more though, renamed to your suggestion.